### PR TITLE
Feat/sharepoint standalone

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/solarsteein`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

## Description

Add standalone sharepoint app 

## Requires 
- [x] https://github.com/ovh/manager/pull/3418/commits/ce88157173a8f9b55c85fd2700b53192e51b210c

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#14432.0 ✘  
* Stage 1
  * e2e -   
  * lint ✔ 
  * SemVer ✘

**Comment by antleblanc** [comment]
@marie-j Could you please resolve existing conflicts? In order to be able to merge it.

Thank you!

**Comment by marie-j** [comment]
> @marie-j Could you please resolve existing conflicts? In order to be able to merge it.
> 
> Thank you!

@antleblanc Done !

**Comment by ovh-cds** [comment]
CDS Report build#14545.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#14545.1 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#14545.2 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#14554.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#14556.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#14556.1 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#14718.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#14728.0 ✘  
* 
  * Build ✘

File: packages/manager/apps/sharepoint/src/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Sharepoint App</title>
```
Comment (antleblanc) [inline packages/manager/apps/sharepoint/src/index.html:5]: **thought:**

```suggestion
        <title>SharePoint App</title>
```

**Review by antleblanc — APPROVED** [review]

**Comment by marie-j** [comment]
Only updated with @antleblanc suggestion

**Pull request merged**